### PR TITLE
feat: add test for target sequencer cert delivery

### PR DESCRIPTION
**Pull request by atanmarko**
# Description

Add test where we check if target subnet sequencer has received certificate with cross chain transaction

Fixes TP-823

## PR Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added or updated tests that comprehensively prove my change is effective or that my feature works

### CODE REVIEW

File: tests/test_cert_sequencer_delivery.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+export TOPOS_CORE_PROXY_CONTRACT_ADDRESS=$(get_topos_core_proxy_contract_address)
+
+
+# Make token transaction on incal network targeting topos network
+# In should be delivered to target topos sequencer
+echo "Executing test of certificate delivery to target topos sequencer..."
+check_artifacts
+topos_subnet_id=$(get_topos_subnet_id)
+receipts_root_hash=$(send_token_with_retry 3 $topos_subnet_id $INCAL_HOST_PORT)
+transaction_valid=$?
```
Comment (JDawg287) [inline tests/test_cert_sequencer_delivery.sh:34]: This is nice and I think we can adapt this logic [here](https://github.com/topos-protocol/local-erc20-messaging-infra/blob/73bfad2f274a162cc2f48f4df07d39d637949236/tests/test_cert_inclusion.sh#L43C75-L43C75) as well. Can we add it to this PR?

Comment (atanmarko) [reply tests/test_cert_sequencer_delivery.sh:34]: You mean to check if certificate with tx is created?

Comment (JDawg287) [reply tests/test_cert_sequencer_delivery.sh:34]: No I mean we don't have the logic to validate the outcome the execution of the `send_token_with_retry` command, in the `test_cert_inclusion` script.
Just so all the tests are unified.

Comment (atanmarko) [reply tests/test_cert_sequencer_delivery.sh:34]: Ok I'll add it

File: tests/test_cert_inclusion.sh [hunk 2]
```diff
@@ -41,7 +41,17 @@ echo "Executing certificate inclusion test..."
 check_artifacts
 incal_subnet_id=$(get_incal_subnet_id)
 receipts_root=$(send_token_with_retry 3 $incal_subnet_id $TOPOS_HOST_PORT)
+transaction_valid=$?
```
[the remark this answers is not in the document]
Comment (atanmarko) [reply tests/test_cert_inclusion.sh:44]: Ok

**Review by JDawg287 — APPROVED** [review]

**Pull request merged**